.github/workflows/ci.yaml

```diff
@@ -0,0 +1,23 @@
+name: CI
```

Comment:
Hi @River-unknown We already have CI setup in circle CI

Don't worry about the CI side of things. I see you've set up an npm script so I'll just add that to our existing CI routine

Reply:
@josephjclark sir shall i then delete this ci.yaml file and then commit?